src/window.rs

```diff
@@ -850,10 +852,14 @@ impl Default for CursorIcon {
     }
 }
 
+/// Fullscreen modes.
 #[derive(Clone, Debug, PartialEq)]
 pub enum Fullscreen {
     Exclusive(VideoMode),
-    Borderless(MonitorHandle),
+
+    /// Providing `None` to `Borderless` will result in fullscreen
+    /// being set on a current monitor.
```

Comment:
```suggestion
    /// Providing `None` to `Borderless` will fullscreen on the current monitor.
```

I think this is a bit better. Since "fullscreen is set on..." doesn't really sound right. And either way it should be "**the** current monitor".